vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -104,7 +104,7 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
                 layer.ipex_fusion = ipex.llm.modules.GatedMLPMOE(
                     layer.w13_weight,
                     layer.w2_weight,
-                    use_prepack=True,
+                    use_prepack=ipex._C.onednn_has_bf16_support(),
```

Comment:
Perhaps make this config as a environment variable is better, such as ```VLLM_CPU_MOE_PREPACK```, by default is ```False```. 

You can refer to ```VLLM_CPU_OMP_THREADS_BIND``` for adding an env in vLLM.

Reply:
I have made the change. I set `True` by default to maintain current behavior, unless you want to change the default to False instead? (Then we probably need to document this flag somewhere e.g. CPU perf)